roofit/roofitcore/src/RooFormulaVar.cxx

```diff
@@ -121,7 +121,7 @@ RooFormulaVar::RooFormulaVar(const RooFormulaVar& other, const char* name) :
   _formExpr(other._formExpr)
 {
   if (other._formula && other._formula->ok()) {
-    _formula.reset(new RooFormula(GetName(), _formExpr, _actualVars));
+    _formula.reset(new RooFormula(GetName(), _formExpr, _actualVars, false));
```

Comment:
```suggestion
    _formula.reset(new RooFormula(GetName(), _formExpr, _actualVars, false));
    _formula.reset(new RooFormula(GetName(), _formExpr, _actualVars, /*checkVariables=*/false));
```

not only helps the reader, but clang-tidy warns if `false`, at some point, stop referring to `checkVariables`! (suggestion applies to the two diffs below this one too)

Reply:
Oh, that's amazing! Didn't know that clang-tidy understands this.